content/posts/dag-van-de-arbeid-looptocht-1-mei-2025/index.en.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
As mentionend on the etherpad, we need to find some way to reference to our chapter. Here I standardised everything to "Tech Workers" as the people from techwerkers.nl, and then "tech workers" (uncapitalized) as "The people that are represented.

In dutch this was "Techwerkers" and "techwerkers" respectively.

Not sure if that just seems weird or not :shrug:. Feel free to adjust however you see fit :sweat_smile: